### PR TITLE
feat: update subscription index and delete

### PR DESCRIPTION
**Pull request by ansmonjol**

### CODE REVIEW

File: src/resources/subscription.yaml [hunk 1]
```diff
@@ -40,6 +40,15 @@ delete:
...
   operationId: destroySubscription
+  parameters:
+    - name: status
+      in: query
+      description: If the field is not defined, Lago will terminate only `active` subscriptions. However, if you wish to cancel a `pending` subscription, please ensure that you include `status=pending` in your request.
+      required: false
+      explode: true
+      schema:
+        type: string
+        example: 'pending'
```
Comment (vincent-pochet) [inline src/resources/subscription.yaml:51]: I think it should have the sub-type "enum":

```yaml
schema:
  type: string
  enum:
    - active
    - pending
  example: 'pending'
```

File: src/resources/subscriptions.yaml [hunk 2]
```diff
@@ -51,6 +51,17 @@ get:
...
+    - name: status
+      in: query
+      description: 'Filter subscriptions by status. Available filter values: `pending`, `canceled`, `terminated`, `active`.'
+      required: false
+      explode: true
+      schema:
+        type: array
+        items:
+          type: string
+        example: '[acive,pending]'
```
Comment (vincent-pochet) [inline src/resources/subscriptions.yaml:63]:
```yaml
items:
  type: string
  enum:
    - pending
    - canceled
    - terminated
    - active
example: [acive,pending]
```

**Review by vincent-pochet — APPROVED** [review]
Two small comments, but looks good :+1:

File: src/resources/subscription.yaml (hunk 1 above)
Comment (ansmonjol) [reply src/resources/subscription.yaml:51]: This attribute only accepts `pending` as a value. So I guess a string do the job right?

File: src/resources/subscriptions.yaml (hunk 2 above)
Comment (ansmonjol) [reply src/resources/subscriptions.yaml:63]: Indeed! Will udpate

Comment (ansmonjol) [reply src/resources/subscriptions.yaml:68]: done

**Pull request merged**